DLplatform/learningLogger.py

```diff
@@ -152,6 +165,21 @@ def logModel(self, filename : str, params: Parameters):
         '''
         pickle.dump(params, open(os.path.join(self._logpath, filename), 'wb'))
 
+
+
+    def logIntermediateModel(self,param: Parameters):
+        '''
+        Logs intermediate model, i.e., saves the parameters of intermediate model
+
+        Parameters
+        ----------
+        params - weights of an averaged model
+        '''
+        filename = 'IntermediateState_' +str(time.time())
```

Comment:
Can you please add the if condition, that this model is saved only when you are in DEBUG mode (it is done also in the model saving that was there before)

Reply:
I have added the If condition to log intermediate models only in DEBUG mode

Reply:
thank you!